lms/static/scripts/frontend_apps/components/BasicLTILaunchApp.js

```diff
@@ -163,17 +167,17 @@ export default function BasicLTILaunchApp() {
 
       return success;
     },
-    [apiSync, authToken, clientRPC]
+    [groupAPICallInfo, authToken, clientRPC]
   );
 
   /**
-   * Fetch the URL of the content to display in the iframe if `viaUrlApi`
-   * exists.
+   * Fetch the URL of the content to display in the iframe if there is Via
```

Comment:
I think we could probably just delete the whole comment, to avoid repeating the code, given that the helper has a pretty obvious name.